arm/Microsoft.Storage/storageAccounts/.parameters/min.parameters.json

```diff
@@ -1,9 +1,5 @@
 {
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
-    "parameters": {
-        "allowBlobPublicAccess": {
-            "value": false
-        }
-    }
+    "parameters": {}
```

Comment:
undo. Otherwise this storage account is flagged by policies

Reply:
I see. Will do